### PR TITLE
Fix: Remove underscores from commands' name

### PR DESCRIPTION
**Pull request by Berbe**

### CODE REVIEW

File: src/console.cpp [hunk 1]
```diff
@@ -221,7 +221,7 @@ static std::string &RemoveUnderscores(std::string &name)
  */
 /* static */ void IConsole::CmdRegister(std::string name, IConsoleCmdProc *proc, IConsoleHook *hook)
 {
-	IConsole::Commands().try_emplace(RemoveUnderscores(std::string(name)), name, proc, hook);
```
Comment (glx22) [inline src/console.cpp:224]: This one is intended so the key is a duplicate of the name.
That way key doesn't contain '_' but displayed list still has them (and matches the `help` command output).

File: src/console.cpp [hunk 2]
```diff
@@ -243,7 +243,7 @@ static std::string &RemoveUnderscores(std::string &name)
  */
 /* static */ void IConsole::AliasRegister(std::string name, std::string cmd)
 {
-	auto result = IConsole::Aliases().try_emplace(RemoveUnderscores(std::string(name)), name, cmd);
```
Comment (glx22) [inline src/console.cpp:246]: Intended too.

File: src/console.cpp [hunk 3]
```diff
@@ -321,7 +321,7 @@ static void IConsoleAliasExec(const IConsoleAlias *alias, byte tokencount, char
 
 						if (param < 0 || param >= tokencount) {
 							IConsoleError("too many or wrong amount of parameters passed to alias, aborting");
-							IConsolePrintF(CC_WARNING, "Usage of alias '%s': %s", alias->name, alias->cmdline);
+							IConsolePrintF(CC_WARNING, "Usage of alias '%s': %s", alias->name.c_str(), alias->cmdline.c_str());
```
Comment (glx22) [inline src/console.cpp:324]: Good catch :)

File: src/console.cpp (hunk 1 above)
Comment (Berbe) [reply src/console.cpp:224]: You cannot use `RemoveUnderscores()` on an `rvalue`, per its signature.

If you need a copy, either:
- Overload `RemoveUnderscores()` to include an object copy definition
- Insert the object 1st and act on it through `RemoveUnderscores`

File: src/console.cpp (hunk 2 above)
Comment (Berbe) [reply src/console.cpp:246]: Same reasoning as before

**Comment by glx22** [comment]
The branch is now merged in master.

**Pull request closed without merging**